portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -61,7 +64,17 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws IOExc
 
     datastore.put(commentEntity);
 
+    Query query = new Query("Comment");
```

Comment:
You should either sort the results, or remove the "import ....SortDirection" line above.

Reply:
Resolving because removed import statement